package.json

```diff
@@ -8,11 +8,21 @@
     "type": "git",
     "url": "git://github.com/woocommerce/woocommerce-gateway-payfast.git"
   },
+  "engines": {
+    "node": ">=12.0.0",
+    "npm": ">=6.9.0"
+  },
   "devDependencies": {
+    "@woocommerce/dependency-extraction-webpack-plugin": "^1.4.0",
+    "@woocommerce/eslint-plugin": "^1.1.0",
+    "@wordpress/scripts": "^13.0.2",
     "node-wp-i18n": "~1.2.3"
   },
   "scripts": {
-    "build": "npm run makepot",
+    "build": "npm run makepot && npm run build:webpack",
+    "build:webpack": "rimraf build/* && wp-scripts build client/blocks/index.js",
+    "start:webpack": "rimraf build/* && wp-scripts start client/blocks/index.js",
```

Comment:
I think there is no need to use `rimraf build/*` because `wp-scripts` [already clean](https://github.com/WordPress/gutenberg/blob/329fe98df660b0da9b3f149ce34fce2bda1b805d/packages/scripts/config/webpack.config.js#L215-L221) the build folder using `clean-webpack-plugin`. What do you think?

Reply:
Good point! Tested and it seems to remove everything from the `build` folder.

Fixed in https://github.com/woocommerce/woocommerce-gateway-payfast/pull/40/commits/d7f6f6de904b68ade581e20590b26f200157611c.